src/test/java/testCases/AccessControlTest.java

```diff
@@ -0,0 +1,96 @@
+package testCases;
+
+import Config.ExtentManager;
+import PageObjects.HomePage;
+import PageObjects.LoginPage;
+import com.aventstack.extentreports.ExtentTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import Constant.Constant;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+public class AccessControlTest {
+
+    ExtentTest extentTest;
+    HomePage homePage;
+    LoginPage loginPage;
+
+    @BeforeMethod
```

Comment:
Create a BaseTest class to hold reusable methods annotated with @BeforeMethod and @AfterMethod when using TestNG; other test classes can then extend it.